### PR TITLE
fix: allow progress bar to be disabled

### PR DESCRIPTION
**Pull request by mgordon-arize**
Add new parameter to llm_classify to allow progress bar format to be customizable and to be disabled

This was because the logs were noisy and verbose does not disable the progress bar

### CODE REVIEW

**Comment by RogerHYang** [comment]
sorry the CI is not working right now. we're still trying to fix it

**Comment by RogerHYang** [comment]
Ok. Please try merge main to update CI. Thanks

**Comment by axiomofjoy** [comment]
@mgordon-arize Can you sign the CLA?

File: packages/phoenix-evals/src/phoenix/evals/executors.py [hunk 1]
```diff
@@ -225,7 +225,10 @@ def termination_handler(signum: int, frame: Any) -> None:
         original_handler = signal.signal(self.termination_signal, termination_handler)
         outputs = [self.fallback_return_value] * len(inputs)
         execution_details = [ExecutionDetails() for _ in range(len(inputs))]
-        progress_bar = tqdm(total=len(inputs), bar_format=self.tqdm_bar_format)
+        if self.tqdm_bar_format is None:
+            progress_bar = tqdm(total=len(inputs), bar_format=self.tqdm_bar_format, disable=True)
+        else:
+            progress_bar = tqdm(total=len(inputs), bar_format=self.tqdm_bar_format)
```
Comment (RogerHYang) [inline packages/phoenix-evals/src/phoenix/evals/executors.py:231]: nit (same for the other location)
```suggestion
        progress_bar = tqdm(
            total=len(inputs), 
            bar_format=self.tqdm_bar_format, 
            disable=self.tqdm_bar_format is None,
        )
```

**Review by RogerHYang — APPROVED** [review]

**Comment by mgordon-arize** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**